### PR TITLE
WIP Review: Minor edits to install-w-machine.

### PR DESCRIPTION
**Pull request by rolfedh**
Fixed typos and minor formatting issues.

### CODE REVIEW

File: docs/install-w-machine.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ Here you use the discovery backend hosted on Docker Hub to create a unique disco
         Status: Downloaded newer image for swarm:latest
         0ac50ef75c9739f5bfeeaf00503d4e6e
 
-    The `docker run` command gets the latest `swarm` image and runs it as a container. The `create` argument makes the Swarm container connect to the Docker Hub discovery service and get a unique Swarm ID, also known as a "discovery token". The token appears in the output, it is not saved to a file on the host. The `--rm` option automatically cleans up the container and removes the file system when the container exits.
+    The `docker run` command gets the latest `swarm` image and runs it as a container. The `create` argument makes the Swarm container connect to the Docker Hub discovery service and get a unique Swarm ID, also known as a "discovery token." The token appears in the output, it is not saved to a file on the host. The `--rm` option automatically cleans up the container and removes the file system when the container exits.
```
Comment (nishanttotla) [inline docs/install-w-machine.md]: I really prefer the British style with punctuation outside the quotation marks, but we can debate that.

**Comment by nishanttotla** [comment]
LGTM

ping @moxiegirl

File: docs/install-w-machine.md [hunk 2]
```diff
@@ -106,7 +106,7 @@ Here, you connect to each of the hosts and create a Swarm manager or node.
         manager   *        virtualbox   Running   tcp://192.168.99.100:2376           v1.9.1
 
 
-2. Your client should still be pointing to Docker Engine on `manager`. Use the following syntax to run a Swarm container as the primary Swarm manager on `manager`.
+2. Your client should still be pointing to Docker Engine on `manager`. Use the following syntax to run a Swarm container as the primary manager on `manager`.
```
Comment (chanwit) [inline docs/install-w-machine.md]: We are running a Swarm manager container here. 

IMHO we need to explicitly saying that we are going to run Swarm manager or Swarm agent as Swarm has 2 roles in a single binary.

Comment (nishanttotla) [inline docs/install-w-machine.md]: I agree, better to be explicit since this tutorial is geared towards new users.

Comment (rolfedh) [inline docs/install-w-machine.md]: @nishanttotla @chanwit There is no ambiguity in the following statement: "to run a Swarm container as the primary manager on `manager`." It says we're running a Swarm container as primary manager.

Comment (nishanttotla) [inline docs/install-w-machine.md]: Fair point. Perhaps we can say something like "to run a Swarm container *that functions* as the primary manager on `manager`"?

Comment (rolfedh) [inline docs/install-w-machine.md]: LGTM!

**Comment by rolfedh** [comment]
@nishanttotla @chanwit Thanks for your contributions! I've updated the file as requested.

**Comment by nishanttotla** [comment]
Thanks @rolfedh. LGTM. We'll have to wait for @moxiegirl to approve too.

**Comment by chanwit** [comment]
Thanks @rolfedh
LGTM

**Comment by vieux** [comment]
@rolfedh can you please squash the 2 last commit, there is still a commit without signature.

**Comment by nishanttotla** [comment]
@vieux do you know why the check for signed commits isn't showing up?

**Comment by rolfedh** [comment]
My @docker.com email address went offline recently. I wonder if that, or updating my signature to my @gmail.com address today had any impact.

**Comment by rolfedh** [comment]
@vieux Could you ask someone with Git-fu resolve the signature issue. I'm concerned about messing things up.

**Comment by vieux** [comment]
@rolfedh @moxiegirl will take care of it

**Comment by nathanleclaire** [comment]
@nishanttotla Not sure the motivation for the change but the DCO check was removed from Travis: https://github.com/docker/swarm/pull/746

**Pull request closed without merging**